### PR TITLE
COMMERCE-9743 - Delivery Subscription is visible on the SF even if it's not enabled, with multiple SKUs product

### PR DESCRIPTION
**Pull request by victorcdp**
Relevant Issues: [COMMERCE-9743](https://issues.liferay.com/browse/COMMERCE-9743), [COMMERCE-9742](https://issues.liferay.com/browse/COMMERCE-9742)

Problem was that there was no check to see if the product's subscriptions are enabled before being shown in the store page.
Proposed fix is to check if they are enabled before being shown.
The COMMERCE-9742 fix is an easy change of display type, so that each subscription is in their own line

### CODE REVIEW

**Comment by victorcdp** [comment]
ci:test:sf

**Comment by victorcdp** [comment]
ci:test:relevant

**Comment by victorcdp** [comment]
ci:test:commerce

**Review by andrea-ale-sbarra — APPROVED** [review]
LGTM

File: modules/apps/commerce/commerce-product-content-web/src/main/resources/META-INF/resources/product_detail/render/view.jsp [hunk 1]
```diff
@@ -147,8 +147,8 @@ long cpDefinitionId = cpCatalogEntry.getCPDefinitionId();
 					/>
 				</c:if>
 
-				<span data-text-cp-instance-subscription-info></span>
-				<span data-text-cp-instance-delivery-subscription-info></span>
+				<span style="display: block" data-text-cp-instance-subscription-info></span>
+				<span style="display: block" data-text-cp-instance-delivery-subscription-info></span>
```
[the remark this answers is not in the document]
Comment (victorcdp) [reply modules/apps/commerce/commerce-product-content-web/src/main/resources/META-INF/resources/product_detail/render/view.jsp:143]: Done @alessiorendina, thank you.

File: modules/apps/commerce/commerce-product-content-web/src/main/resources/META-INF/resources/product_detail/render/view.jsp [hunk 2]
```diff
@@ -139,8 +139,8 @@ long cpDefinitionId = cpCatalogEntry.getCPDefinitionId();
 					/>
 				</c:if>
 
-				<span data-text-cp-instance-subscription-info></span>
-				<span data-text-cp-instance-delivery-subscription-info></span>
+				<span style="d-block" data-text-cp-instance-subscription-info></span>
+				<span style="d-block" data-text-cp-instance-delivery-subscription-info></span>
```
Comment (alessiorendina) [inline modules/apps/commerce/commerce-product-content-web/src/main/resources/META-INF/resources/product_detail/render/view.jsp:143]: Don't use the `style` property. Use `class` instead.

**Review by alessiorendina — CHANGES_REQUESTED** [review]
@victorcdp Please check my comment, thanks!

**Comment by victorcdp** [comment]
@alessiorendina changed as requested. Thank you

**Review by alessiorendina — APPROVED** [review]

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:relevant

**Comment by alessiorendina** [comment]
ci:test:commerce-functional

**Comment by Brunno0liveira** [comment]
Resent at https://github.com/liferay-commerce/liferay-portal/pull/4444

**Comment by Brunno0liveira** [comment]
ci:close

**Pull request closed without merging**